scripts/json2db.pl

```diff
@@ -159,6 +163,13 @@
                          . DBI->errstr . "\n";
                  }
             }
+                if ( $motion->{motion}->{policy_vote} =~ /3/ ) {
+                    my $pw_id = "public_whip_dreammp" . $dreamid . "_has_strong_vote";
+                    my $has_strong = $strong_vote_check->execute( $pw_id, $mp_id_num );
+                    if ( $strong_vote_check->rows() < 1 ) {
+                        $strong_vote_add->execute( $pw_id, 1, $mp_id_num);
```

Comment:
Does this need to use strong_vote_update at some point, for if there are now no longer any strong votes?